charts/app-config/values-production.yaml

```diff
@@ -1450,6 +1450,11 @@ govukApplications:
       uploadAssets:
         enabled: false
       workerEnabled: true
+      redis:
+        enabled: true
+        redisUrlOverride:
```

Comment:
Re: [my confusion on the other branch](https://github.com/alphagov/govuk-helm-charts/pull/2248#discussion_r1704190581), if you've got the energy to do so, it could be worth specifying in the commit message that although we're provisioning the new instance with `enabled: true`, the addition of the manual URL override is what ensures we're not switching to it yet.

Reply:
Good idea, I've updated the commit message here.